website/docs/basics/helpers-javascript.md

```diff
@@ -0,0 +1,311 @@
+
+**Usage:**
+
+```js
+  import { dynamicImport } from '@eightshift/frontend-libs/scripts/helpers';
+
+  dynamicImport(require.context('./../../custom', true, /assets\/index.js$/));
+```
+
+## escape-string
+
+Takes provided string and removes special characters. Characters that will be removed: `([;&,.+*~':"!^#$%@[\]()=>|]`.
+
+* @param string $key Key to check.
+
+**Usage:**
+
+```js
+  import { escapeString } from '@eightshift/frontend-libs/scripts/helpers';
+
+  escapeString.escapeString('Special string');
+```
+
+## responsiveSelectors
+
+Create responsive selectors used for responsive attributes.
+
+* @param array   items        Array of breakpoints.
+* @param string  selector     Selector for this breakpoint.
+* @param string  parent       Parent block selector.
+* @param boolean use_modifier If false, you can use this selector for visibility.
+
+**Usage:**
+
+```js
+  import { responsiveSelectors } from '@eightshift/frontend-libs/scripts/helpers';
+
+  responsiveSelectors($attributes['width'], 'width', $blockClass);
+```
+
+**Output:**
+
+```js
+  .block-column__width-large--4
+```
+
+## selectorB
+
+Return's BEM selector for HTML class and checks if block is set.
+
+* @param string block BEM Block selector. *(required)*
+* @param string element BEM Element selector.
+* @param string modifier BEM Modifier selector.
+
+**Usage:**
+
+```js
+  import { selectorB } from '@eightshift/frontend-libs/scripts/helpers';
+
+  selectorB(blockClass);
+  selectorB(blockClass, selectorClass);
+  selectorB(blockClass, selectorClass, modifierClass);
+```
+
+**Equivalent:**
+
+```js
+  blockClass ? `${blockClass}` : '';
+  blockClass ? `${blockClass}__${selectorClass}` : '';
+  blockClass ? `${blockClass}__${selectorClass}--${modifierClass}` : '';
+```
+
+
+## selectorE
+
+Return's BEM selector for HTML class and checks if the element is set.
+
+* @param string block BEM Block selector. *(required)*
+* @param string element BEM Element selector. *(required)*
+* @param string modifier BEM Modifier selector.
+
+**Usage:**
+
+```js
+  import { selectorE } from '@eightshift/frontend-libs/scripts/helpers';
+
+  selectorE(blockClass, selectorClass);
+  selectorE(blockClass, selectorClass, modifierClass);
+```
+
+**Equivalent:**
+
+```js
+  selectorClass ? `${blockClass}__${selectorClass}` : '';
+  selectorClass ? `${blockClass}__${selectorClass}--${modifierClass}` : '';
+```
+
+## selectorM
+
+Return's BEM selector for HTML class and checks if the modifier is set.
+
+* @param string block BEM Block selector. *(required)*
+* @param string element BEM Element selector. *(required)*
+* @param string modifier BEM Modifier selector. *(required)*
+
+**Usage:**
+
+```js
+  import { selectorM } from '@eightshift/frontend-libs/scripts/helpers';
+
+  selectorM(blockClass, selectorClass, modifierClass);
+```
+
+**Equivalent:**
+
+```js
+  modifierClass ? `${blockClass}__${selectorClass}--${modifierClass}` : '';
+```
```

Comment:
Do we really need this? Since `selectorB` does the same thing (depending on the number of parameters.) We can call that one `bemSelectors`.